pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -19,14 +20,16 @@ import (
 )
 
 type BlackboxExporter struct {
-	Client        client.Client
-	Log           logr.Logger
-	Ctx           context.Context
-	BlackBoxImage string
+	Client                 client.Client
+	Log                    logr.Logger
+	Ctx                    context.Context
+	BlackBoxImage          string
+	BlackBoxNamespacedName types.NamespacedName
```

Comment:
```suggestion
	Image          string
	NamespacedName types.NamespacedName
```
Isn't it redundant to have BlackBox in those variable names since `BlackboxExporter` is in the type already?

Reply:
Thanks @NautiluX , I changed this.